vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,93 @@
+#include "vpd_tool.hpp"
+
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{
+            "VPD Command line tool to dump the inventory and to read and "
```

Comment:
Why are we limiting the use case here?
Do we need to mention the use cases in the description?